From: Tomas Henzl <thenzl@redhat.com>
Date: Sun, 29 Aug 2010 15:49:38 -0400
Subject: [block] cciss: fix scsi status typo
Message-id: <1283097002-3341-40-git-send-email-thenzl@redhat.com>
Patchwork-id: 27896
O-Subject: [RHEL6 PATCH 39/63] cciss: fix scsi status typo
Bugzilla: 568830
RH-Acked-by: Neil Horman <nhorman@redhat.com>

fix scsi status value.  It does not need to be shifted.

diff --git a/drivers/block/cciss_scsi.c b/drivers/block/cciss_scsi.c
index ae4169a..51b5096 100644
--- a/drivers/block/cciss_scsi.c
+++ b/drivers/block/cciss_scsi.c
@@ -611,7 +611,7 @@ complete_scsi_command( CommandList_struct *cp, int timeout, __u32 tag)
                         			cp,  
 						ei->ScsiStatus); 
 #endif
-					cmd->result |= (ei->ScsiStatus < 1);
+					cmd->result |= ei->ScsiStatus;
                 		}
 				else {  /* scsi status is zero??? How??? */
 					
